src/seedu/addressbook/commands/SortCommand.java

```diff
@@ -0,0 +1,38 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.data.person.ReadOnlyPerson;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Sort the list of persons
+ */
+
+public class SortCommand extends Command {
+    public static final String COMMAND_WORD = "sort";
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Displays all persons in the address book sorted by names.\n"
+            + "Example: " + COMMAND_WORD;
+     @Override
+     public CommandResult execute() {
+         List<ReadOnlyPerson> allPersons = addressBook.getAllPersons().immutableListView();
+         List<ReadOnlyPerson> sortedPersons = new ArrayList<ReadOnlyPerson>();
+         int i,j;
+         for(i=0;i<allPersons.size();i++){
+             sortedPersons.add(allPersons.get(i));
+             }
+         int n=allPersons.size();
+         for(i=0;i<n-1;i++){
```

Comment:
Is this the best way to sort the list. Can you think of a better way to do this, eg. use of Java in built comparator etc. Refer to your peers' code if you are not sure.

Reply:
Do I need to do all these enhancements in another PR? Or only one feature in the new PR will do?

Reply:
Thank you